src/doc/en/developer/coding_basics.rst

```diff
@@ -353,7 +353,9 @@ information. You can use the existing functions of Sage as templates.
 
        - ``n`` -- integer
 
-       - ``p`` -- prime integer (default: `2`); coprime with ``n``
+       - ``p`` -- prime integer (default: `2`); coprime with `n`
```

Comment:
```suggestion
       - ``p`` -- prime integer (default: ``2``); coprime with `n`
```

Reply:
or no backticks at all?

Reply:
In sage, input values are often mathematical. This line shows an example: integer 2.

Reply:
like n in "coprime with n".

Reply:
If that is the case I will need to revert my change in #38068.

Why isn't 
- ``n`` -- integer
written as
- `n` -- integer?

(with single instead of double backticks)

Reply:
The thing before `--` denotes the input argument (code snippet). 
```
The input argument ``n`` denotes an integer `n`.
```

Reply:
Shouldn't the default value also be considered as a code snippet?

Reply:
The default value is a value of the input argument. The value is often mathematical in sage. Then we want to show it as math. We may even
```
``m`` -- matrix (default: identity matrix)
```
or
```
``m`` -- matrix (default: `I`)
```

Reply:
Ok, I think it is indeed better to write everything after `--` in the nicest way available (math for math).

Reply:
I merged #38068 here, made the suggested changes: https://github.com/sagemath/sage/pull/38068/files#issuecomment-2142935336

Thanks.